x-pack/plugins/ml/public/components/anomalies_table/anomaly_details.js

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+/*
+ * React component for displaying details of an anomaly in the expanded row section
+ * of the anomalies table.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import _ from 'lodash';
+
+import {
+  EuiDescriptionList,
+  EuiIcon,
+  EuiLink,
+  EuiSpacer,
+  EuiText
+} from '@elastic/eui';
+import { formatDate } from '@elastic/eui/lib/services/format';
+
+import { EntityCell } from './entity_cell';
+import {
+  getSeverity,
+  showActualForFunction,
+  showTypicalForFunction
+} from 'plugins/ml/../common/util/anomaly_utils';
+import { formatValue } from 'plugins/ml/formatters/format_value';
+
+const TIME_FIELD_NAME = 'timestamp';
+const INFLUENCERS_LIMIT = 5;
```

Comment:
nit: this is defined in the same way in `anomaly_table.js`, it could be moved to a common constants file or passed down in the component tree from the parent.